app/assets/javascripts/select-kit/addon/components/category-drop.js

```diff
@@ -149,16 +179,23 @@ export default ComboBoxComponent.extend({
         parentCategoryId = -1;
       }
 
-      const results = (
-        await Category.asyncSearch(filter, {
-          parentCategoryId,
-          includeUncategorized: this.siteSettings.allow_uncategorized_topics,
-          includeAncestors: true,
-          limit: 15,
-        })
-      ).categories;
+      const result = await Category.asyncSearch(filter, {
+        parentCategoryId,
+        includeUncategorized: this.siteSettings.allow_uncategorized_topics,
+        includeAncestors: true,
+        // Show all categories if possible (up to 18), otherwise show just
+        // first 15 and let CategoryDropMoreCollection show the "show more" link
+        limit: 18,
+      });
+
+      const categories =
+        result.categoriesCount > 15
```

Comment:
I may be misunderstanding, but shouldn't this be `> 18`?

Reply:
That is correct. Thank you!